src/database.ts

```diff
@@ -55,24 +60,102 @@ import {
   TransactionRunner,
 } from './transaction-runner';
 
-export interface GetDatabaseOptions {
+import {google as spanner_client} from '../proto/spanner';
+import {Any, Schema} from './common';
+import {ServiceError, CallOptions} from 'grpc';
+import {Readable, Transform} from 'stream';
+import {Stats} from 'fs';
+import {StreamProxy} from 'google-gax/build/src/streamingCalls/streaming';
+
+interface CreateBatchTransactionCallback {
+  (
+    err?: Error | null,
```

Comment:
We shouldn't be using `Error`, gRPC Errors also have `status`, `metadata` and `details` fields, please use the appropriate gRPC error type.